### PR TITLE
Bug 1179712 - Adding pin card component r=benfrancis

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0ea5d8c447fc1ab5636c996782f7632e2ca05c0b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6f129f53915d79360c33ec86c0578366fc3f52dc

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=09a01b443844470c48ad1986779e1de0c2984168

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f60da420f79fb99f26072d6ccfede553d0da12b6

File: shared/elements/gaia_pin_card/script.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    this.bgElement = this._template.querySelector('.background');
+    this.descElement = this._template.querySelector('.description');
+    this.titleElement = this._template.querySelector('header');
+
+    shadow.appendChild(this._template);
+
+    ComponentUtils.style.call(this, baseurl);
+  };
+
+  proto.__defineSetter__('background', function(background) {
```
Comment (KevinGrandon) [inline shared/elements/gaia_pin_card/script.js]: Please use ```defineProperty``` here instead. Example: https://github.com/mozilla-b2g/gaia/blob/master/shared/elements/gaia_switch/script.js#L73

File: shared/elements/gaia_pin_card/style.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
+
+.pin-card {
+  position: relative;
+  float: left;
+  min-width: 14rem;
+  height: 10.5rem;
+  border-radius: 5px;
```
Comment (KevinGrandon) [inline shared/elements/gaia_pin_card/style.css]: Nit: use 0.5rem to be consistent.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3464ebc5d322dd92bdab02be3010882163ce9766

**Pull request merged**